src/eradiate/experiments/_dem.py

```diff
@@ -235,9 +200,9 @@ def _context_kwargs(self) -> dict[str, t.Any]:
 
         for measure in self.measures:
             if measure_inside_atmosphere(self.atmosphere, measure):
-                kwargs[
-                    f"{measure.sensor_id}.atmosphere_medium_id"
-                ] = self.atmosphere.medium_id
+                kwargs[f"{measure.sensor_id}.atmosphere_medium_id"] = (
+                    self.atmosphere.medium_id
```

Comment:
Is this good coding style? I suppose this happens because of the line length limit, no?

Reply:
This is because, I think, the pr-ecommit hook for formatting now uses Ruff instead of Black. The two styles are very similar, but not identical.